db/fixtures/notification_types.yml

```diff
@@ -374,3 +374,8 @@
   :expires_in: 7.days
   :level: success
   :audience: user
+- :name: generic_superadmin_notification
+  :message: '%{subject}: %{message}'
+  :expires_in: 7.days
+  :level: success
```

Comment:
I realize this is generic, but wondering if this should be a warning (Is warning even possible?)

Reply:
yeah, I opened this PR to ask questions like this.  Good question.